src/components/FAQSection/FAQSection.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import DynamicListSection from "../DynamicListSection";
+import FAQs from "./FAQs.js";
+
+const GetThreeFAQs = (faqs) => {
+  return faqs.slice(0, 3);
+};
+
+const showThreeFAQs = GetThreeFAQs(FAQs);
+
+const FAQSection = () => {
```

Comment:
I recommend to rename FAQSection to something more descriptive like FrequentlyAskedQuestionsSection

Reply:
Yes, I will correct.